README.md

```diff
@@ -62,6 +62,10 @@ An overview and detailed descriptions of the implemented operators can be found
 
 ## Setup
 
+### Windows
```

Comment:
I would move this note as last point of this section, right before "Gradoop modules" begins.